tests/conftest.py

```diff
@@ -81,7 +83,12 @@
 except ImportError:
     _STREAMLIT_INSTALLED = False
 
-# All fixture functions have prefix 'fixture_' and explicitly declared name so they
+# Hypothesis settings
+settings.register_profile("ci", deadline=500, max_examples=200)
+if strtobool(os.getenv("CI", "false")):
```

Comment:
Please double-check that "CI" needs to be in caps here @AdrianSosic .

Reply:
Yes, this is indeed correct. From the Github docs:
<img width="767" alt="image" src="https://github.com/emdgroup/baybe/assets/23265127/2cb60160-b70b-4aa6-b2b4-9e49ca7ae577">

In fact, used this already somewhere else in the past